### PR TITLE
Add `transformers_int4_npu_pipeline_win` in all-in-one benchmark

### PR DESCRIPTION
**Pull request by ch1y0q**
### 4. How to test?
- [ ] N/A
- [ ] Unit test: Please manually trigger the PR Validation [here](https://github.com/intel-analytics/ipex-llm-workflow/actions/workflows/llm-PR-validation.yml) by inputting the PR number (e.g., `1234`). And paste your action link here once it has been successfully finished.
- [ ] Application test
- [ ] Document test
- [ ] ...

### 5. New dependencies

- [ ] New Python dependencies
       - Dependency1 
       - Dependency2
       - ...
- [ ] New Java/Scala dependencies and their license
       - Dependency1 and license1
       - Dependency2 and license2
       - ...

### CODE REVIEW

File: python/llm/dev/benchmark/all-in-one/run.py [hunk 1]
```diff
@@ -680,6 +682,81 @@ def transformers_int4_npu_win(repo_id,
...
+                                                     quantization_group_size=npu_group_size, transpose_value_cache=transpose_value_cache,
+                                                     use_cache=True, attn_implementation="eager").eval()
+        tokenizer = LlamaTokenizer.from_pretrained(model_path, trust_remote_code=True)
+    else:
+        model = AutoModelForCausalLM.from_pretrained(model_path, load_in_low_bit=low_bit, trust_remote_code=True, pipeline=True, torch_dtype=torch.float16,
+                                                     optimize_model=optimize_model, max_context_len=max_context_len, max_prompt_len=int(in_out_len[0]), 
+                                                     quantization_group_size=npu_group_size, transpose_value_cache=transpose_value_cache,
+                                                     use_cache=True, attn_implementation="eager").eval()
+        tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
+    end = time.perf_counter()
```
Comment (rnwang04) [inline python/llm/dev/benchmark/all-in-one/run.py:723]:
```python
if repo_id in CHATGLM_IDS:
     model = AutoModel.from_pretrained(model_path, load_in_low_bit=low_bit, trust_remote_code=True, pipeline=True,
                                      optimize_model=optimize_model, max_context_len=max_context_len, max_prompt_len=int(in_out_len[0]),
                                      quantization_group_size=npu_group_size, transpose_value_cache=transpose_value_cache,
                                      torch_dtype=torch.float16, attn_implementation="eager").eval()
    tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
elif repo_id in LLAMA_IDS:
    model = AutoModelForCausalLM.from_pretrained(model_path, load_in_low_bit=low_bit, trust_remote_code=True, pipeline=True, torch_dtype=torch.float16,
                                                 optimize_model=optimize_model, max_context_len=max_context_len, max_prompt_len=int(in_out_len[0]), 
                                                 quantization_group_size=npu_group_size, transpose_value_cache=transpose_value_cache,
                                                 use_cache=True, attn_implementation="eager").eval()
    tokenizer = LlamaTokenizer.from_pretrained(model_path, trust_remote_code=True)
else:
    model = AutoModelForCausalLM.from_pretrained(model_path, load_in_low_bit=low_bit, trust_remote_code=True, pipeline=True, torch_dtype=torch.float16,
                                                 optimize_model=optimize_model, max_context_len=max_context_len, max_prompt_len=int(in_out_len[0]), 
                                                 quantization_group_size=npu_group_size, transpose_value_cache=transpose_value_cache,
                                                 use_cache=True, attn_implementation="eager").eval()
    tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
```
->

```python
model = AutoModelForCausalLM.from_pretrained(model_path, load_in_low_bit=low_bit, trust_remote_code=True, pipeline=True, torch_dtype=torch.float16,
                                             optimize_model=optimize_model, max_context_len=max_context_len, max_prompt_len=int(in_out_len[0]), 
                                             quantization_group_size=npu_group_size, transpose_value_cache=transpose_value_cache,
                                             use_cache=True, attn_implementation="eager").eval()
tokenizer = AutoTokenizer.from_pretrained(model_path, trust_remote_code=True)
```

**Comment by rnwang04** [comment]
Please fix the code dtyle.

**Review by rnwang04 — APPROVED** [review]
others LGTM

**Review by Oscilloscope98 — APPROVED** [review]
Maybe also update the README.md in all-in-one folder for updated config.yml
Others LGTM

File: python/llm/dev/benchmark/all-in-one/run.py (hunk 1 above)
Comment (rnwang04) [reply python/llm/dev/benchmark/all-in-one/run.py:712]: Also update this for `transformers_int4_npu_win`

Comment (Oscilloscope98) [reply python/llm/dev/benchmark/all-in-one/run.py:712]: Maybe also update this for `transformers_int4_npu_win`?

Comment (rnwang04) [reply python/llm/dev/benchmark/all-in-one/run.py:712]: Please update this in next PR. I want to merge this PR first for test.

**Pull request merged**